docs/iot/get-started/tizen-studio-install.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Just question. Where is this tizen-studio-install.md file used? Couldn't find it from iot/toc_all.md.

Reply:
This file is not used anymore. When I searched developer.tizen.org, this file was detected. I'll restore it because we don't need to update it. 
I will remove all unused files later.